instrumentation/mongodb/README.md

```diff
@@ -0,0 +1,35 @@
+# brave-instrumentation-mongodb
+
+This includes [`TraceMongoCommandListener`](src/main/java/brave/mongodb/TraceMongoCommandListener.java), a
+[`CommandListener`](https://mongodb.github.io/mongo-java-driver/3.12/driver/reference/monitoring/#command-monitoring)
+for the Mongo Java driver that will report via Brave how long each command takes, along with relevant tags like the
+collection/view name, the command's name (`insert`, `update`, `find`, etc.).
+
+This instrumentation can only be used with the synchronous MongoDB driver. Do not use it with the asynchronous or
+reactive drivers as tracing data will be incorrect.
+
+## Span properties:
+- `name`: command name (and collection/view name, if available). Examples: `find myCollection`, `listCollections`, etc.
+- `kind`: `CLIENT`
+- `remoteServiceName`: `mongodb-${databaseName}`. Example: `mongodb-myDatabase`.
+- `remoteIpAndPort`: the IP address and port number of the MongoDB server that the command was issued to
+- `error`: `Throwable` in case of failed command
+- Tags:
+  - `mongodb.command.name`: the name of the MongoDB command. Examples: `find`, `listCollections`, etc.
+  - `mongodb.command`: a possibly truncated, JSON version of the full command. Example:
+    `{"find": "myCollection", "filter": { "myField": { "$gte": 5 } } }`"
+  - `mongodb.collection`: the name of the MongoDB collection/view that the command operates on, if available
+  - `mongodb.cluster.id`: a client-generated identifier that uniquely identifies a connection to a MongoDB cluster
```

Comment:
unless we expect a value tagged about cluster that is not id, I'd drop the .id part

Reply:
I left this as `mongodb.cluster_id` to be (mostly) aligned with [MongoMetricsCommandListener](https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/mongodb/MongoMetricsCommandListener.java#L65).

Reply:
RATIONALE.md this :) always good to have citations of homework you did